core/src/style/polylineStyle.cpp

```diff
@@ -177,12 +178,10 @@ void PolylineStyle::buildLine(const Line& _line, const DrawRule& _rule, const Pr
         } else { height = extrude[1]; }
     }
 
-    GLfloat layer = _props.getNumeric("sort_key") + params.order;
```

Comment:
Nope, tangram-js doesn't use "sort_key" at all unless specified in the
scene file

On Wed, Oct 14, 2015 at 3:50 PM, Varun <notifications@github.com> wrote:

> In core/src/style/polylineStyle.cpp
> <https://github.com/tangrams/tangram-es/pull/344#discussion_r42043376>:
>
> > @@ -177,12 +178,10 @@ void PolylineStyle::buildLine(const Line& _line, const DrawRule& _rule, const Pr
> >          } else { height = extrude[1]; }
> >      }
> >
> > -    GLfloat layer = _props.getNumeric("sort_key") + params.order;
>
> So not using sort_key values for ordering of roads now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/tangram-es/pull/344/files#r42043376>.
>